### PR TITLE
English documentation of importers

### PR DESCRIPTION
**Pull request by cutterkom**
added english translation to the documentation of importers with the help of deepl.com

### CODE REVIEW

**Comment by mateuswetah** [comment]
Thank you soooooo much for this @cutterkom! It's great to see someone improving our English documentation. I'll review it later today and give some feedback 😃

File: importers.md [hunk 1]
```diff
@@ -32,69 +32,69 @@ O **Importador CSV** permite que usuários importem itens para uma coleção dir
...
-* `special_attachments` - permite que o usuário informe os anexos. Consulte [Importando arquivos e anexos](#importar-arquivos-e-anexos).
-* `special_comment_status` - permite que o usuário informe se os items podem receber ou não comentários, as opções são: *"open"* ou *"closed"* (padrão).
+The special columns that can be used are:
 
+* `special_item_status` - Informs about the status of the item. Possible values are:
+  * `draft`
+  * `private`
+  * `publish`
+* `special_item_id` - The **item ID** in the Tainacan database. This function is useful for re-importing items and allowing you to decide whether to update existing items or ignore them and add new items.
+* `special_document` - allows you to enter the item document. See [importing files and attachments](#importar-arquivos-e-anexos).
```
Comment (mateuswetah) [inline importers.md]: Here, as you renamed the section a few lines bellow (79) you can refer its slug version, in this case `#import-files-and-attachments`

Comment (mateuswetah) [reply importers.md]: Oh nevermind, you did on the other commit :)

File: importers.md [hunk 2]
```diff
@@ -121,89 +121,105 @@ Imagens enviadas por FTP,myfolder/image.jpg||myfolder/image2.jpg
...
-Por exemplo:
+If one of your CSV columns has values for a taxonomy metadata and this taxonomy has a hierarchy, you can enter that hierarchy using the `>>` sign.
+
+Example:
 
 ```
-nome, Categoria
-Nome do item,Categoria Pai>>Categoria Filha>>Categoria Neta
+name, category
+name of item,parent category>>child category>>needed category
```
Comment (mateuswetah) [inline importers.md:160]: Here and in the line 169, I believe it its "grandchild category"  instead of "needed category"

File: importers.md [hunk 3]
```diff
@@ -121,89 +121,105 @@ Imagens enviadas por FTP,myfolder/image.jpg||myfolder/image2.jpg
...
+    
+    
 7. Selecione ou crie uma **Coleção Destino** para indicar onde os itens serão criados;
    *. Selecionando criar uma nova coleção em branco, ao `Concluir` a coleção, você será redirecionado para o **Importador** novamente.
+   
+   
+   
+   
+   
+8. Select or create a **Fate Collection** to indicate where the items will be created
```
Comment (mateuswetah) [inline importers.md]: the meaning of "Coleção Destino" here is to indicate that this is the collection where the items will be imported to. I believe earlier you mentioned it as "Destination Collection", which seems to make more sense :)

Comment (mateuswetah) [reply importers.md]: Ok it seems that this one was solved in the following commit, thanks :+1:

**Comment by cutterkom** [comment]
I am glad, that you like it. 

Thanks for all your work! Tainacan is a great piece of software for collections

**Comment by mateuswetah** [comment]
Thanks again @cutterkom. Feel free to perform future translations and if you need any help regarding the usage of Tainacan, we will be glad to help here, in the WordPress Forum or in our community mail list: https://lists.riseup.net/www/arc/tainacan/

**Pull request merged**